colossus/src/main/scala/colossus/protocols/memcache/Memcache.scala

```diff
@@ -226,12 +249,17 @@ sealed trait MemcacheWriteCommand extends MemcacheCommand {
     b.append(SP)
 
     b.append(dataSizeStr)
+
+    if (casUniqueStr.nonEmpty) {
+      b.append(SP)
```

Comment:
SP should be named MemcachedMessageDelimeter or something along the line. SP is short for space ?? Not very helpful